From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@yossidahan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

mimig1 committed:
Cosmos DB: DocumentDB API -> SQL API

mimig1 committed:
Fixed formatting

mimig1 committed:
wording update

mimig1 committed:
updates

carolinacmoravia committed:
C5398: LOCBLOCKER - Extra spaces causing content blocked for localiza… #25836

Hello, @Mareat,
There are extra spaces causing content blocked for localization. Could you help to merge this PR, please?
Many thanks.

cwatson-cat committed:
removed manage class url

BryanLa committed:
updates

BryanLa committed:
updates

mimig1 committed:
more updates

BryanLa committed:
updates

BryanLa committed:
updates

PRMerger10 committed:
Merge pull request #29336 from BryanLa/bryanla-release-msi-preview

Preview notice

v-alje committed:
Bringing even with master.

mimig1 committed:
more updates

v-alje committed:
Bringing even with master.

BryanLa committed:
remove note

TomArcherMsft committed:
putting tutorial on landing page

PRMerger14 committed:
Merge pull request #29463 from BryanLa/bryanla-release-msi-preview

remove note

mimig1 committed:
updates

mimig1 committed:
fixing issues

mimig1 committed:
Acrolinx fixes to bring scores above 80

v-alje committed:
Bringing even with master.

mimig1 committed:
fix validation errors

mimig1 committed:
QA pass

tfitzmac committed:
revising multiple instance

mimig1 committed:
Merge branch 'master' into 11292017docdbtosql

mimig1 committed:
link fix

v-alje committed:
Bringing even with master.

CamSoper committed:
rewrite and a new Core version

mimig1 committed:
reverted date change

mimig1 committed:
Merge branch 'master' into 11292017docdbtosql

v-alje committed:
Bringing even with master.

tfitzmac committed:
revising

kgremban committed:
Add ML tutorial to IoT Edge landing page

CamSoper committed:
Incorporated feedback

kuhussai committed:
pre-review

BryanLa committed:
revive article

BryanLa committed:
updates

BryanLa committed:
updates

rwike77 committed:
added info

v-alje committed:
Bringing even with master.

BryanLa committed:
initial populate

BryanLa committed:
initial populate

cwatson-cat committed:
Upd to Windows VM topics to remove stray classic portal references

cwatson-cat committed:
Removed references in VM includes to classic portal and manage.windowsazure.com

kuhussai committed:
Updated based on pre-review

kuhussai committed:
updated based on pre-review

BryanLa committed:
review feedback updates

cwatson-cat committed:
upd chef topic to raise score

wmgries committed:
Fix up misplaced scale target.

BryanLa committed:
add TOC

wmgries committed:
Fix up of @jeffpatt24 contribution.

cwatson-cat committed:
Upd VM include file to raise score

mattbriggs committed:
changing author to mattbriggs

danieldobalian committed:
Changing link to docs

mattbriggs committed:
resolve merge conflicts

Commit:
KRIS_azure portal meta data

ggailey777 committed:
Moving video down in the topic

This is per agreements we have with Kris.

BryanLa committed:
updates

mattbriggs committed:
updating metadata

PRMerger16 committed:
Merge pull request #29727 from kriscrider/meta-portal-12.11

KRIS_azure portal meta data

PRMerger17 committed:
Merge pull request #29728 from ggailey777/patch-4

Moving video down in the topic

mattbriggs committed:
resovle conflict merge

Commit:
Merge pull request #29596 from cwatson-cat/12-5-rmvClassic-ChinaDevGuide

Update China Dev Guide to remove classic portal reference

tynevi committed:
Merge pull request #29550 from mimig1/11292017docdbtosql

11292017docdbtosql

BryanLa committed:
updates

GitHubber17 committed:
Merge pull request #29721 from mattbriggs/mdb_changeauthor

changing author

wmgries committed:
Cleanup of @klaaslanghout contribution.

kgremban committed:
Fixed small error in C# tutorial

BryanLa committed:
add article to TOC

wmgries committed:
Fix stray smart quote.

mimig1 committed:
Cosmos DB: Updated Graph doc author values

BryanLa committed:
updates

mimig1 committed:
update

alkohli committed:
Software version 2.1.1.518 or later.

Feedback from Vijay

daveba committed:
changing author value

GitHubber17 committed:
Merge pull request #29659 from BryanLa/bryanla-adal-errorhandling

Revive ADAL error handling article

PRMerger15 committed:
Merge pull request #29730 from kgremban/dec11-fix

Fixed small error in C# tutorial

daveba committed:
adding redirect to new source article

GitHubber17 committed:
Merge pull request #29493 from TomArcher/terraform-put-tutorial-on-landing-page

Putting tutorial on Terraform landing page

GitHubber17 committed:
Merge pull request #29678 from cwatson-cat/12-9-VMRmvClassicPortRef-Windows

Upd to Windows VM topics to remove stray classic portal references

GitHubber17 committed:
Merge pull request #29679 from cwatson-cat/12-9VMRmvClassicPortRef-includes

Removed references in VM includes to classic portal

BryanLa committed:
updates

PRMerger14 committed:
Merge pull request #29732 from mimig1/12112017redirects

Cosmos DB: Updated Graph doc author values

daveba committed:
deleting article and removing reference to it in TOC

PRMerger10 committed:
Merge pull request #29206 from carolinacmoravia/patch-497

C5398: LOCBLOCKER - Extra spaces causing content blocked for localiza…

GitHubber17 committed:
Merge pull request #29723 from danieldobalian/patching-link

Changing link to docs

PRMerger14 committed:
Merge pull request #29734 from alkohli/patch-162

Software version 2.1.1.518 or later.

sethmanheim committed:
Updates per PM feedback, also new screen shots

kgremban committed:
Fixed typo and moved glossary in TOC

sethmanheim committed:
Acrolinx

GitHubber17 committed:
Merge pull request #29729 from mattbriggs/mdb_changeauthtessa

updating metadata

Commit:
Added TIMESTAMP BY OVER info

BryanLa committed:
updates

sethmanheim committed:
Code fixes for consistency

GitHubber17 committed:
Merge pull request #29731 from wmgries/scale-targets-fix

Fix storage scale targets and misc

tynevi committed:
Merge pull request #29577 from CamSoper/aspnet-storage

rewrite and a new Core version

kuhussai committed:
updated to clear warning in PR #29653

GitHubber17 committed:
Merge pull request #29735 from MicrosoftDocs/remove-classic-portal-article-active-directory-self-service-application-access

remove-classic-portal-article-active-directory-self-service-application-access

kuhussai committed:
updated to clear warning in PR #29653

kuhussai committed:
updated to clear warning in PR#29653

PRMerger15 committed:
Merge pull request #29663 from rwike77/freshness

added info

kuhussai committed:
updated for new storage-account-options.md page

GitHubber17 committed:
Merge pull request #29646 from kgremban/dec8-landingpage

Add ML tutorial to IoT Edge landing page

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

BryanLa committed:
updates

rwike77 committed:
updated link

Ja-Dunn committed:
Merge pull request #29737 from kgremban/dec11-TOC

Fixed typo and moved glossary in TOC

Lisaco88 committed:
Merge pull request #29670 from BryanLa/bryanla-release-msi-populate

Bryanla release msi initial populate

PRMerger14 committed:
Merge pull request #29747 from rwike77/freshness

updated link

kuhussai committed:
added minor edits

sethmanheim committed:
One more code fix

kuhussai committed:
Fixed "0ptions" to "Options" in title

kuhussai committed:
lowercase "options" in title

SoubhagyaDash committed:
Merge pull request #4 from MicrosoftDocs/master

merge with upstream

SoubhagyaDash committed:
updated availability tests count in the AI limits doc

Ja-Dunn committed:
Merge pull request #29736 from sethmanheim/wk-sbqs

Updates per PM feedback, also new screen shots

JasonWHowell committed:
Update scenario-document-collection-analysis.md

tynevi committed:
Revert "Bryanla release msi initial populate"

PRMerger14 committed:
Merge pull request #29751 from SoubhagyaDash/master

Updated Availability tests limit to 100

Ja-Dunn committed:
Merge pull request #29653 from kuhussai/master

pre-review

Commit:
Merge pull request #29752 from JasonWHowell/patch-3

Update scenario-document-collection-analysis.md

tynevi committed:
Merge pull request #29753 from MicrosoftDocs/revert-29670-bryanla-release-msi-populate

Revert "Bryanla release msi initial populate"

spelluru committed:
Fixed the broken link

nitinme committed:
freshness updates

tfitzmac committed:
added examples

v-thepet committed:
fix broken link

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

mahesh-unnikrishnan committed:
Remove classic portal references.

PRMerger15 committed:
Merge pull request #29757 from nitinme/nitinme-freshness-updates

Freshness updates

Commit:
Merge pull request #29756 from spelluru/fixlink1211

Fixed the broken link

Ja-Dunn committed:
Merge pull request #29548 from tfitzmac/1207multiple

revising multiple instance

PRMerger15 committed:
Merge pull request #29760 from mahesh-unnikrishnan/patch-15

Remove classic portal references.

v-alje committed:
Merge pull request #29758 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #29759 from v-thepet/newfix

fix broken link

Commit:
Merge pull request #29738 from JSeb225/patch-24

Added TIMESTAMP BY OVER info
